Server/routes/authRoute.js

```diff
@@ -8,12 +8,14 @@ const {
   recoveryRequestController,
   validateRecoveryTokenController,
   resetPasswordController,
+  deleteUserController
 } = require("../controllers/authController");
 
 //Auth routes
 router.post("/register", registerController);
 router.post("/login", loginController);
 router.post("/user/:userId", verifyJWT, userEditController);
+router.delete("/user/:userId", deleteUserController);
```

Comment:
This route is unsecured, as it is right now anyone can delete any user without even being logged in.

We need to apply two middleware functions to this route:

1.  `verifyJWT` -  This middleware ensures the user is logged in with a valid JWT
2.  `verifyOwnership` - This ensures that the user with the JWT owns the resource they are trying to access.  This middleware will need some modification before it works with the `User` document.

Reply:
I just passed the `verifyJWT` similar to `line 17`, making it like this:
`router.delete("/user/:userId", verifyJWT, deleteUserController);`
Didn't add the `verifyOwnership` based on what you said on need for modifications.